server/src/server.ts

```diff
@@ -71,34 +91,58 @@ class Server {
     this._db = pgp(connectionString);
   }
 
-  private _configServer (): void {
+  private __configServer (): void {
     this._port = process.env.PORT || Config.PORT;
     this._app.use(cors());
     this._app.use(bodyParser.urlencoded({ extended: true }));
     this._app.use(bodyParser.json());
   }
 
-  private _configFrontendRoutes (): void {
+  /**
+   * SERVICES
+   */
+  private _services (): void {
+    // init user service
+    this._userService = new UserService(this._db);
+    // init auth service
+    this._authService = new AuthService(this._userService);
+  }
+
+  /**
+   * ROUTES
+   */
+  private _routes (): void {
+    this.__frontendRoutes();
+    this.__apiRoutes();
+  }
+
+  private __frontendRoutes (): void {
     // serve frontend from server/dist/public
     this._app.use(express.static(path.join(__dirname, "public")));
   }
 
-  private _configApiRoutes (): void {
+  private __apiRoutes (): void {
     // set routes to paths
-    this._app.use("/api/v1/users", this.__usersRoute());
-    this._app.use("/api/v1/logs", this.__logsRoute());
+    this._app.use("/api/v1/auth", this.___authRoute());
+    this._app.use("/api/v1/users", this.___usersRoute());
+    this._app.use("/api/v1/logs", this.___logsRoute());
   }
 
-  private __usersRoute (): express.Router {
+  private ___authRoute (): express.Router {
+    // create new router
+    let router: express.Router = express.Router();
+    // init and return auth route
+    return new AuthRouteV1(this._authService, router).createRoutes();
+  }
+
+  private ___usersRoute (): express.Router {
     // create new router
     let router: express.Router = express.Router();
-    // init user service
-    const userService: UserService = new UserService(this._db);
     // init and return users route
-    return new UsersRouteV1(userService, router).createRoutes();
+    return new UsersRouteV1(this._userService, router).createRoutes();
   }
 
-  private __logsRoute (): express.Router {
+  private ___logsRoute (): express.Router {
```

Comment:
I really can accept the use of one _ since the tooling of js / ts is as mature as in java or so but what is the purpose of two or even three underscores??? This is really hard to read and search.